examples/pylint/C0113_uneeded_not.py

```diff
@@ -0,0 +1,14 @@
+"""pylint: uneeded not
+
+"""
+
+def is_true():
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    temp = 5
+    if temp is not not True:
```

Comment:
I don't like showing a comparison between different types. Maybe use `if not not temp > 3`